glottolog3/models.py

```diff
@@ -541,6 +541,43 @@ class LanguoidSchema(Schema):
     def make_languoid(self, data):
         return Languoid(**data)
 
+def validate_id_type(type):
+    VALID_TYPES = [
+        IdentifierType.iso.value,
+        IdentifierType.glottolog.value,
+        IdentifierType.wals.value,
+        IdentifierType.ethnologue.value,
+        'name'
+    ]
+
+    if not type in VALID_TYPES:
+        raise ValidationError('Invalid identifier type: {}'.format(type))
+
+class IdentifierSchema(Schema):
+    id = fields.Str(default = '')
+    name = fields.Str(required=True)
+    type = fields.Str(required=True, validate=validate_id_type)
+    description = fields.Str(default = '')
```

Comment:
Leave as None instead of empty string

Reply:
The default in the postgres database sets it to empty string as well, is there a reason we'd prefer None over the empty string?

Reply:
I agree it doesn't really matter here, but in general we should leave things that don't exist as None so that if we try to manipulate them or check their existence, it gives an accurate answer. redundancy should be avoided anyway!